test/features/deploy_quarkus_service.feature

```diff
@@ -146,14 +149,16 @@ Feature: Deploy quarkus service
 
     Then GraphQL request on Data Index service returns ProcessInstances processName "orders" within 2 minutes
 
+    @test
     Examples:
-      | native   | minutes |
-      | disabled | 10      |
+      | native   | minutes | build-request-cpu | build-limit-cpu | build-request-memory |
+      | disabled | 10      | 1                 | 4               | 4Gi                  |
 
     @native
+    @test
     Examples:
-      | native  | minutes |
-      | enabled | 20      |
+      | native   | minutes | build-request-cpu | build-limit-cpu | build-request-memory |
```

Comment:
Seems like the native with persistence and events is resources demanding. I put the same values as for onboarding